src/Components/v2/CollectionsHubsHomepageNav.tsx

```diff
@@ -14,16 +14,10 @@ interface CollectionsHubsHomepageNavProps {
 }
 
 export const CollectionsHubsHomepageNav = track(
-  {
-    context_page: AnalyticsSchema.PageName.HomePage,
```

Comment:
I'm not sure how accurately I understand the tracking hooks, so I raise this as more of a question than a suggestion.

I _think_ we still want to leave lines 17 to 20. 

My understanding of how the `track(...)` function works here is that any properties we set in this object will be used for any calls to `trackEvent` inside this function. The `context_page`, `context_module`, and `subject` all seem important enough to apply to all tracked events in this component. 

By removing these lines, I _think_ that the event tracked on line 46 would end up sending blank values for each of those properties, which I don't think is what we want.

Does that sound correct, or do you think my understanding of the tracking helpers is incorrect?